test/email/email.integration.spec.ts

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
I am curious why this set of tests is not located in any of the modules (emails, for example)?)